commands/command.gd

```diff
@@ -112,6 +112,12 @@ var command_description:String :
 	set(value): return
 	get: return _get_description()
 
+## The color of this command's text
+## Used by the Comment command to fade the text out
+var command_color:Color :
```

Comment:
I think this need to be  `command_text_color`

Reply:
done